cli/command/formatter/node.go

```diff
@@ -12,7 +14,7 @@ import (
 )
 
 const (
-	defaultNodeTableFormat           = "table {{.ID}} {{if .Self}}*{{else}} {{ end }}\t{{.Hostname}}\t{{.Status}}\t{{.Availability}}\t{{.ManagerStatus}}"
+	defaultNodeTableFormat           = "table {{.ID}} {{if .Self}}*{{else}} {{ end }}\t{{.Hostname}}\t{{.Status}}\t{{.Availability}}\t{{.ManagerStatus}}{{if .TLSStatus}}\t{{.TLSStatus}}{{end}}"
```

Comment:
Do we want this to be shown by default in `docker node ls`? Or only in `docker node inspect`? Discussing with @ehazlett and @cpuguy83 and we think it may not be needed; but happy to hear about that